app/assets/stylesheets/custom.scss

```diff
@@ -98,3 +113,66 @@ footer {
     }
   }
 }
+
+/* sidebar */
+aside {
+  section.user_info {
+    margin-top: 20px;
+  }
+  section {
+    padding: 10px 0;
+    margin-top: 20px;
+    &:first-child {
+      border: 0;
+      padding-top: 0;
+    }
+    span {
+      display: block;
+      margin-bottom: 3px;
+      line-height: 1;
+    }
+    h1 {
+      font-size: 1.4em;
```

Comment:
add one empty line for every block, please

Reply:
done